commercetools-models/src/main/java/io/sphere/sdk/carts/ItemShippingTarget.java

```diff
@@ -12,6 +12,6 @@
 
     String getAddressKey();
 
-    int getQuantity();
+    long getQuantity();
```

Comment:
We should document this breaking change in our release notes. And additionally we always use non primitive types even for mandatory fields, because this wouldn#t cause any problems if we make the optional later.

Reply:
Totally makes sense 👍 
will include